src/components/CallView/shared/VideoBackground.vue

```diff
@@ -21,10 +21,14 @@
 
 <template>
 	<div class="video-backgroundbackground">
+		<ResizeObserver
```

Comment:
Can we avoid having the resize imported there? I'm not sure it scales well.
If you are in a call with 25 people we watch 25 elements for resize all the time,
but we could also just trigger it when the grid is updated?

Reply:
I don't think it can be avoided, do you have something in mind?
Also notice that this is not a grid-only problem: it affects every video component

Reply:
Ok, I moved the logic for the grid videos to the grid component and left the resize observer only for the "big" background.